build_rules.gradle

```diff
@@ -1253,6 +1254,125 @@ ext.applyGrpcNature = {
 
 /*************************************************************************************************/
 
+ext.applyPortabilityNature = {
+  println "applyPortabilityNature with " + (it ? "$it" : "default configuration") + " for project $project.name"
+  applyJavaNature(enableFindbugs: false, enableErrorProne: false, shadowClosure: {
+    // guava uses the com.google.common and com.google.thirdparty package namespaces
+    relocate "com.google.common", "org.apache.beam.vendor.guava.v20.com.google.common"
```

Comment:
Where do these version strings come from? If they need to be maintained, can you add some notes/validation?

Reply:
Added comment explaining the process.